k9mail/src/main/res/layout/account_setup_incoming.xml

```diff
@@ -243,11 +284,12 @@
                     android:contentDescription="@string/account_setup_incoming_compression_label"
                     />
             </LinearLayout>
-            <View
+            <!--<View
```

Comment:
Almost all these xmls has this view. What is it for? It seemed that nothing happened after I removed them.